### PR TITLE
 Added: Contribution Guideline #14 

### PR DESCRIPTION
**Pull request by darshitdudhaiya**

### CODE REVIEW

**Comment by darshitdudhaiya** [comment]
@FahimFBA I Added Guidelines into your project please merge it.

File: README.md [hunk 1]
```diff
@@ -189,3 +189,42 @@ In dark mode!
 
 ## :page_with_curl: License
 This project is licensed under the MIT License - see the [LICENSE](LICENSE) file for details.
+
+
+## Instructions to run
```
Comment (FahimFBA) [inline README.md:194]: The instructions are not necessary as that is already present in details in the README.

**Review by FahimFBA — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (darshitdudhaiya) [reply README.md:189]: the changes are applied

File: README.md [hunk 2]
```diff
@@ -186,6 +186,40 @@ In dark mode!
...
 
 
+## Instructions to run
+```
+$ git clone https://github.com/FahimFBA/StudentBook.git
+$ cd StudentBook
+$ npm install
+$ npm run dev
+```
+
```
Comment (FahimFBA) [inline README.md:196]: You have still kept the instructions to clone, installing dependencies and so on.

**Review by FahimFBA — CHANGES_REQUESTED** [review]

**Review by FahimFBA — APPROVED** [review]

**Comment by FahimFBA** [comment]
Thank you for your contribution. 👍

**Comment by FahimFBA** [comment]
Sorry, your pull request #16 can't be accepted as I needed to revert the changes in pull request #17

**Pull request merged**